### PR TITLE
Extensibility + changelogs

### PR DESCRIPTION
**Pull request by johnlafleur**
Text within docs.

### CODE REVIEW

**Review by sherifnada — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 [![](docs/.gitbook/assets/airbyte_white-background%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%281%29.svg)](https://airbyte.io)
 
-### Data integration made simple, secure and reliable.
+### Data integration made simple, secure and extensible.
```
Comment (jrhizor) [inline README.md:7]:
```suggestion
### Data integration made simple, secure, and extensible.
```

File: README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
 
-The new open-source standard to sync data from applications & databases to warehouses.
+The new open-source standard to sync data from applications, APIs & databases to warehouses.
```
Comment (jrhizor) [inline README.md:9]:
```suggestion
The new open-source standard to sync data from applications, APIs, and databases to warehouses.
```

File: docs/changelog.md [hunk 3]
```diff
@@ -4,6 +4,35 @@ description: Be sure to not miss out on new features and improvements!
...
+* our own **Snowflake** warehouse integration
+* **10 additional Singer taps** 
+
+## 0.3.0 - expected around 10/23/2020
+
+Here is what we have in mind:
+
+* new guides \(how to build scratch/singer\) 
+* our own **BigQuery** warehouse integration 
+* our own **PostGres** warehouse integration
```
Comment (jrhizor) [inline docs/changelog.md:22]:
```suggestion
* our own **Postgres** warehouse integration 
```

**Review by jrhizor — APPROVED** [review]
Just a couple of typo fixes. Looks good.

File: README.md [hunk 4]
```diff
@@ -15,8 +15,8 @@ The new open-source standard to sync data from applications & databases to wareh
 Airbyte is on a mission to make data integration pipelines a commodity.
 
 * **Maintenance-free connectors you can use in minutes**. Just authenticate your sources and warehouse, and get connectors that adapt to schema and API changes for you.
-* Designed to **cover the long tail of integrations**, as Airbyte will be active in maintaining the pipelines’ reliability. 
-* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, by offering scheduling and orchestration. 
+* **Building new integrations made trivial.** We make it very easy to add new integrations that you need, using the language of your choice, by offering scheduling and orchestration.
```
Comment (michel-tricot) [inline README.md:18]:
```suggestion
* **Building new integrations made trivial.** We make it easy to build and add the integrations that you need, using the language of your choice, by offering scheduling and orchestration. 
```

**Review by michel-tricot — APPROVED** [review]
Did you do it with gitbook?

**Comment by michel-tricot** [comment]
Test notification for John

**Pull request merged**